### PR TITLE
ER-1158 DfE Frontend update

### PR DESCRIPTION
**Pull request by martikat**
[ER-1158](https://dfedigital.atlassian.net/browse/ER-1158)

### CODE REVIEW

File: app/components/header_component.html.slim [hunk 1]
```diff
@@ -21,6 +19,8 @@
       .dfe-header__menu
         button#toggle-menu.dfe-header__menu-toggle aria-controls='header-navigation' aria-expanded='false' data-action='click->reveal#toggle'
           | Menu
+  = tag.div(class: 'dfe-width-container dfe-header__service-name')
```
Comment (peterdavidhamilton) [inline app/components/header_component.html.slim:22]: Why use Ruby for the wrapper?

File: app/assets/stylesheets/application.scss [hunk 2]
```diff
@@ -5,18 +5,12 @@
 // https://design.education.gov.uk/design-system/dfe-frontend
 // https://design-system.service.gov.uk/styles/colour/
 
-// https://github.com/alphagov/govuk_frontend_toolkit/blob/master/stylesheets/_colours.scss
-
 
 $govuk-new-link-styles: true;
 $govuk-page-width: 1200px;
```
Comment (peterdavidhamilton) [inline app/assets/stylesheets/application.scss:10]: Is this needed?

File: package.json [hunk 3]
```diff
@@ -3,16 +3,16 @@
   "packageManager": "yarn@4.0.2",
   "scripts": {
     "build": "esbuild app/javascript/*.* --bundle --sourcemap --outdir=app/assets/builds --public-path=/assets",
-    "build:css": "sass ./app/assets/stylesheets/application.scss:./app/assets/builds/application.css --no-source-map --load-path=node_modules --quiet-deps",
+    "build:css": "sass app/assets/stylesheets/application.scss app/assets/builds/application.css --no-source-map --load-path=node_modules --quiet-deps",
```
Comment (peterdavidhamilton) [inline package.json:6]: Has the CLI changed?

File: app/components/header_component.html.slim (hunk 1 above)
Comment (martikat) [reply app/components/header_component.html.slim]: I have updated this so it is just the div and class on this line

File: app/assets/stylesheets/application.scss (hunk 2 above)
Comment (martikat) [reply app/assets/stylesheets/application.scss]: I have put this back in as it does have some useful information

File: package.json (hunk 3 above)
Comment (martikat) [reply package.json:6]: This matches with how CDT has been setup/updated

Comment (peterdavidhamilton) [reply package.json:6]: The colon to space separation caught my eye.

**Review by peterdavidhamilton — CHANGES_REQUESTED** [review]

**Comment by peterdavidhamilton** [comment]
@martikat there are deprecation warnings due to assets we don't have. References to things we don't need or can't find should be removed rather than errors be silenced.
```
DEPRECATION WARNING: The asset "images/govuk-mask-icon.svg" is not present in the asset pipeline.
Falling back to an asset that may be in the public folder.
This behavior is deprecated and will be removed.
To bypass the asset pipeline and preserve this behavior,
use the `skip_pipeline: true` option.
 (called from index at /home/runner/work/early-years-foundation-reform/early-years-foundation-reform/app/controllers/pages_controller.rb:9)
```

**Review by peterdavidhamilton — CHANGES_REQUESTED** [review]
In case you didn't compare with CDT, the last commit was an opportunity to use a partial so the head was consistent and future changes happen in one place. Could we copy that please.

**Review by peterdavidhamilton — APPROVED** [review]

**Pull request merged**